### ISSUE TITLE
Train with different size and Inference with different size.

### ISSUE THREAD

**Issue by bayesian-mind** [BichenWuUCB/squeezeDet#131]
@BichenWuUCB I have a few questions as I was playing around with squeezedet - 
1. Is it possible using the squeezedet architecture to train the network on size of let's say 321x321x3 and inference on image of size 561x561x3? Why do the size of training and inference have to be same?
2. I was trying to train the network with single class, but I am getting `nan` class loss? I tried training with higher batch size and lower learning rate but that didn't solve the problem either.
